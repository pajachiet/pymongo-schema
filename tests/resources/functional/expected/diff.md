
### Database: test_db2
|Hierarchy     |Previous Schema        |New Schema            |
|--------------|-----------------------|----------------------|
|              |test_db2               |None                  |


### Database: test_db1
#### Collection: test_col2 
|Hierarchy     |Previous Schema        |New Schema            |
|--------------|-----------------------|----------------------|
|              |test_col2              |None                  |

#### Collection: test_col3 
|Hierarchy     |Previous Schema        |New Schema            |
|--------------|-----------------------|----------------------|
|              |None                   |test_col3             |

#### Collection: test_col1 
|Hierarchy     |Previous Schema        |New Schema            |
|--------------|-----------------------|----------------------|
|              |borough                |None                  |
|              |None                   |_borough              |
|cuisine       |{"type": "string"}     |{"type": "ARRAY"}     |
|grades        |grade                  |None                  |

