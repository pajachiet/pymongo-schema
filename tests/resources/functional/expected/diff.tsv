Database	Collection	Hierarchy	Previous Schema	New Schema
test_db2			test_db2	
test_db1	test_col2		test_col2	
test_db1	test_col3			test_col3
test_db1	test_col1		borough	
test_db1	test_col1			_borough
test_db1	test_col1	cuisine	{"type": "string"}	{"type": "ARRAY"}
test_db1	test_col1	grades	grade	
